src/braket/aws/aws_quantum_task.py

```diff
@@ -501,7 +503,10 @@ def _(
         )
 
     create_task_kwargs.update(
-        {"action": circuit.to_ir().json(), "deviceParameters": device_parameters.json()}
+        {
+            "action": circuit.to_ir(ir_type=IRType.OPENQASM).json(),
```

Comment:
How do we determine the `SerializationProperties` for `to_ir()` ?

Reply:
The default, virtual, is used. It is set [here](https://github.com/aws/amazon-braket-sdk-python/blob/feature/openqasm_generation/src/braket/circuits/serialization.py#L38). I believe that's the correct choice, since every device supports virtual qubits, but the local simulator does not support physical qubits.

Reply:
I think we might want to use physical qubits in certain cases (for eg. when disableQubitRewiring is set).